LICENSE.txt

```diff
@@ -1,5 +1,5 @@
 Copyright (c) 2015-2016 Cryptonomex Inc. <contact@cryptonomex.com>
-Copyright (c) 2015-2017 contributors <CONTRIBUTORS.txt>
```

Comment:
- We have 2 lines of Copyright ?
- We have a reference to CONTRIBUTORS.txt witch is empty.

I think we can do something more like: https://github.com/bitshares/bitshares-ui/blob/develop/LICENSE.md for the header.